Makefile

```
@@ -330,7 +340,7 @@ manifests: $(CONTROLLER_GEN))
 
 .PHONY: generate-protobuf
 generate-protobuf: # generates protobuf
```

Can we also setup up the binaries here required for the target?

```suggestion
generate-protobuf: $(PROTOC) $(PROTOC_GEN_GO) # generates protobuf
```

---

ah nice!